outputs/client.go

```diff
@@ -14,6 +14,7 @@ import (
 	"regexp"
 	"strings"
 
+	functions "cloud.google.com/go/functions/apiv1"
```

Comment:
```suggestion
	gcpfunctions "cloud.google.com/go/functions/apiv1"
```
I prefer this naming to avoid confusion

Reply:
done